org.eclipse.xtext.ui.codetemplates/src-gen/org/eclipse/xtext/ui/codetemplates/services/CodetemplatesGrammarAccess.java

```diff
@@ -160,15 +161,16 @@
 		private final RuleCall cTemplateBodyParserRuleCall_2 = (RuleCall)cGroup.eContents().get(2);
 		private final Keyword cLessThanSignLessThanSignKeyword_3 = (Keyword)cGroup.eContents().get(3);
 		
-		//TemplateBodyWithQuotes TemplateBody hidden():
+		//TemplateBodyWithQuotes TemplateBody hidden ():
```

Comment:
@ArneDeutsch Do we miss a formatting rule for the hidden tokens clause? The space shouldn't be there, should it?